add codec selection with setCodecPreferences

fippo opened the pull request:
because munging is bad and we didn't have any other example. One might
still have to fall back to munging but these samples are not a guide to
SDP munging

alvestrand commented:
I'd prefer to clone this into a new demo rather than cluttering up pc1.
once we start cluttering, there's no end to the clutter we might end up with. One demo, one feature.

alvestrand commented:
we've started down the slippery slope with sdpSemantics; if you clone it to pc_bells_and_whistles, you can take that one along :-)

fippo committed:
add codec selection sample

using
  https://w3c.github.io/webrtc-pc/#dom-rtcrtptransceiver-setcodecpreferences

fippo commented:
now someone has to fix apprtc. I nominate @henbos :-p

alvestrand commented on src/content/peerconnection/change-codecs/index.html:
```diff
@@ -0,0 +1,78 @@
+        Started With WebRTC</a>.</p>
+
+
+    <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/peerconnection/pc1"
```
Oops - fix the source link...
